docs/source/top_level_elements/name.rst

```diff
@@ -0,0 +1,382 @@
+
+    <https://example.org/objects/1>
+        relators:cre "Daniel, Charles R. (Charlie), Jr., 1930-" .
+
+Names with empty URIs
+---------------------
+
+There are some records with the namePart "Bemis Bro. Bag Company".  The matches would all be apart of this set:
+
+
+.. code-block:: python
+
+    ['volvoices:2495', 'volvoices:2496', 'volvoices:2497', 'volvoices:2498', 'volvoices:2499', 'volvoices:2500', 'volvoices:2501', 'volvoices:2502', 'volvoices:2503', 'volvoices:2504', 'volvoices:2505', 'volvoices:2506', 'volvoices:2507', 'volvoices:2508', 'volvoices:2509', 'volvoices:2510', 'volvoices:2511', 'volvoices:2512', 'volvoices:2513', 'volvoices:2455', 'volvoices:2456', 'volvoices:2457', 'volvoices:2477', 'volvoices:2478', 'volvoices:2479', 'volvoices:2480', 'volvoices:2481', 'volvoices:2482', 'volvoices:2483', 'volvoices:2484', 'volvoices:2485', 'volvoices:2486', 'volvoices:2467', 'volvoices:2468', 'volvoices:2469', 'volvoices:2470', 'volvoices:2471', 'volvoices:2472', 'volvoices:2473', 'volvoices:2474', 'volvoices:2475', 'volvoices:2476', 'volvoices:2458', 'volvoices:2459', 'volvoices:2461', 'volvoices:2462', 'volvoices:2463', 'volvoices:2464', 'volvoices:2465', 'volvoices:2466', 'volvoices:2487', 'volvoices:2488', 'volvoices:2489', 'volvoices:2490', 'volvoices:2491', 'volvoices:2492', 'volvoices:2493', 'volvoices:2494']
+
+We should fix these and add the actual valueURI.
+
+Names with @usage="primary"
+---------------------------
+
+There are 314 records with an @usage="primary" attribute.  An example is
+`kefauver:150412002 <https://digital.lib.utk.edu/collections/islandora/object/kefauver%3A150412002/datastream/MODS>`_.
+
+.. code-block:: xml
+
+    <name usage="primary">
+        <namePart>unknown</namePart>
+        <role>
+            <roleTerm authority="marcrelator" type="text" valueURI="http://id.loc.gov/vocabulary/relators/cre">Creator</roleTerm>
+        </role>
+    </name>
+
+We should drop this.
+
+Namepart with @type="date"
+--------------------------
+
+There are 10,370 records with a namePart with a @type="date".  This value seems to relate to the birth and death dates of
+the person.
+
+An example is `egypt:230 <https://digital.lib.utk.edu/collections/islandora/object/egypt%3A230/datastream/MODS>`_
+
+If we need to keep this data, we need to have an external triple store and decide how to model names.
+
+Name Descriptions
+-----------------
+
+We have 10,309 records with a name with a description.  This is normally a nationality.  Here is the exhaustive list:
+
+.. code-block:: python
+
+    [None, 'American', 'Argentine', 'Austrian', 'Brazilian', 'British', 'Canadian', 'Danish', 'Dutch', 'Finnish', 'Flemish', 'French', 'German', 'Greek', 'Italian', 'Japanese', 'Norwegian', 'Spanish', 'Swiss', 'Turkish']
+
+An example is `archivision:4817 <https://digital.lib.utk.edu/collections/islandora/object/archivision%3A4817>`_.
+
+If we need to keep this data, we need to have an external triple store and decide how to model names.
```

Comment:
@markpbaggett I vaguely recall having a conversation about working towards leveraging/parsing values from a given valueURI in this type of case. Do we need to refer to that here (just a brief mention) or should that articulation live in the mapping document?

Reply:
@CanOfBees, honestly, I don't remember what specifically was agreed upon.  Are you saying we drop this but potentially leverage from an associated valueURI?

Reply:
@markpbaggett apologies - you're right: I think we talked about dropping this data and pulling appropriate information from the URI.

Reply:
@CanOfBees does this address your request?

Reply:
@markpbaggett Yes - thanks for adding it.